tests/python/relay/test_pass_convert_op_layout.py

```diff
@@ -1473,29 +1557,30 @@ def expected():
 
 
 if __name__ == "__main__":
-    test_qnn_binary_no_convert_layout()
-    test_no_convert_layout()
-    test_conv_convert_layout()
-    test_conv_nhwc_convert_layout()
-    test_conv_bias_pool_convert_layout()
-    test_conv_concat_convert_layout()
-    test_dual_path_convert_layout()
-    test_bn_convert_layout()
-    test_slice_like_convert_layout()
-    test_resnet_convert_layout()
-    test_scalar_convert_layout()
-    test_conv_bn_convert_layout()
-    test_qnn_conv_requantize_convert_layout()
-    test_qnn_conv_concat_convert_layout()
-    test_qnn_conv_add_convert_layout()
-    test_qnn_conv_nhwc_convert_layout()
-    test_conv_convert_kernel_layout()
-    test_conv_transpose_convert_layout()
-    test_conv_roi_align_convert_layout()
-    test_conv_roi_pool_convert_layout()
-    test_conv_strided_slice_convert_layout()
-    test_deformable_conv_bias_pool_convert_layout()
-    test_default_keyword()
-    test_different_ops_convert_layout()
-    test_no_desired_layout()
-    test_convert_with_config()
+    # test_qnn_binary_no_convert_layout()
+    # test_no_convert_layout()
+    # test_conv_convert_layout()
+    # test_conv_nhwc_convert_layout()
+    # test_conv_bias_pool_convert_layout()
+    # test_conv_concat_convert_layout()
+    # test_dual_path_convert_layout()
+    # test_bn_convert_layout()
+    # test_slice_like_convert_layout()
+    test_transpose_convert_layout()
+    # test_resnet_convert_layout()
+    # test_scalar_convert_layout()
+    # test_conv_bn_convert_layout()
+    # test_qnn_conv_requantize_convert_layout()
+    # test_qnn_conv_concat_convert_layout()
+    # test_qnn_conv_add_convert_layout()
+    # test_qnn_conv_nhwc_convert_layout()
+    # test_conv_convert_kernel_layout()
+    # test_conv_transpose_convert_layout()
+    # test_conv_roi_align_convert_layout()
+    # test_conv_roi_pool_convert_layout()
+    # test_conv_strided_slice_convert_layout()
+    # test_deformable_conv_bias_pool_convert_layout()
+    # test_default_keyword()
+    # test_different_ops_convert_layout()
+    # test_no_desired_layout()
+    # test_convert_with_config()
```

Comment:
Did you forget to uncomment these tests?

Reply:
Ah yeah. Thanks for pointing out.